server/validation/campaign.js

```diff
@@ -0,0 +1,32 @@
+const User = require("../models/user");
+const Campaign = require("../models/campaign");
+
+exports.validateCampaignInput = async data => {
+  const errors = {};
+  const { name, ownedBy } = data;
+
+  if (!name) errors.name = "name is required";
+
+  const userExists = await User.findById(ownedBy);
```

Comment:
you probably don't need to check this, since you are using the authenticated user, instead you should check if req.user.id and ownedBy are the same id, and if not , throw a forbidden status code

Reply:
Got it. Fixed.